taccsite_cms/static/site_cms/css/src/_imports/tools/x-drop-cap.css

```diff
@@ -1,19 +1,12 @@
 .x-drop-cap {
-  /* Drop the letter */
-  float: left;
   /* Estimate horizontal spacing to match magazine design */
   /* WARN: Firefox calculates `margin-…` differently here, so use `padding-…` */
   padding-right: 0.125em; /* approx: `1em` of normal text */
 
   font-size: 6em; /* approx: 3 lines, of normal text, tall */
 
-  /* (Firefox v73) Nudge letter up until it meets top of normal text */
-  /* CAVEAT: Also has effect in Safari and Chrome/Edge, but not enough */
-  padding-top: 0.0625em;
-  /* (Safari v13, Chrome/Edge v80) Nudge letter up more until it meets top of normal text */
-  /* MAGIC: Firefox does not need this */
-  /* SEE: https://bugzilla.mozilla.org/show_bug.cgi?id=1233271 */
-  /* MAGIC: Has no effect on Firefox */
-  /* SEE: https://bugzilla.mozilla.org/show_bug.cgi?id=290125 */
-  line-height: 0.65;
+  /* Not yet available on Firefox */
+  initial-letter: 3;
+  /* Support for Safari */
+  -webkit-initial-letter: 3;
```

Comment:
**Hm.** Safari uses a serif font. Not awful... Have you found any solution or bug report?

Reply:
Possible known issue: https://caniuse.com/css-initial-letter
<img width="1376" alt="Screenshot 2023-12-15 at 3 45 30 PM" src="https://github.com/TACC/Core-CMS/assets/63771558/d1680ed9-1cce-470e-8dcc-8089f75093f9">

Reply:
K. Safari 9 is too old. No concern.

> Safari implementation is incomplete. Does not allow applying web fonts to the initial letter.

And that explains why the CMS's web font is not used. I haven't thought of a workaround. This bug can stay. If M.S. vomits, we'll just drop `-webkit-initial-letter`.